invenio_records_resources/records/api.py

```diff
@@ -10,10 +10,12 @@
 """Records API."""
 
 import mimetypes
+import os
 from contextlib import contextmanager
 
 from invenio_db import db
 from invenio_files_rest.models import FileInstance, ObjectVersion
+from invenio_files_rest.utils import FALLBACK_MIMETYPE
```

Comment:
this import is very prone to compatibility issues - I find it a bit risky to import variables from utils of another module (normally I would expect it to be exposed as config, not hidden in utils, if it is supposed to be used by other modules)